### ISSUE TITLE
YOLO v2 link does not open.

### ISSUE THREAD

**Issue by leejaymin** [onnx/models#258]
Recently, Yolo-v2 model was uploaded. But the link does not open.
Is there anyone who can download this model?

**Comment by prasanthpul** [comment]
How did you download it? Many of our models are storing in Git LFS. If you are using the command line, you'll need to make sure Git LFS is installed. If you are using browser, you can go to https://github.com/onnx/models/blob/master/vision/object_detection_segmentation/yolov2/model/yolov2.onnx and click Download

**Comment by leejaymin** [comment]
@prasanthpul Thank you for letting me know the download method. Finally, I got the model of Yolo-v2. However, I think that this model is not correct because Reoge layer is represented by 4 maxpool layers in ONNX format. For verification, I ran this model with onnx_runtime. However, output tensor is not transformed into proper boundings box and probabilities.

I would like to know whether this model was verified before merging into master branch.

**Comment by prasanthpul** [comment]
@doughtmw

**Comment by doughtmw** [comment]
The YOLOv2 model has the same output format as the [TinyYOLOv2 model](https://github.com/onnx/models/tree/master/vision/object_detection_segmentation/tiny_yolov2) in this repo. It is designed that way to allow for compatibility with ONNX opsets 7, 8. 

The input tensor `(3 x 416 x 416)` is formatted to an output tensor `(425 x 13 x 13)` which then requires further processing to extract bounding boxes - just like the TinyYoloV2 model. [Here is a sample from Microsoft](https://docs.microsoft.com/en-us/dotnet/machine-learning/tutorials/object-detection-onnx) showing how to format the TinyYOLOv2 model; this can be easily modified to work with the full YOLOv2 model by changing the anchors and class labels (now have 80 COCO classes) to account for the differently sized tensor output.

**Comment by leejaymin** [comment]
@doughtmw Thank you for the quick response and kind guidance. As you mentioned, I took account of further processing to extract the right bounding boxes. In my case, onnx runtime and [yolov2-pytorch](https://github.com/marvis/pytorch-yolo2) are used. After post-processing, I did not get any bounding boxes with your model. 

Unlike your model, if I use the original darknet model and CFG, I got the correct bounding boxes on [yolov2-pytorch](https://github.com/marvis/pytorch-yolo2). I guess that reorg layer is not well represented by ONNX operators.
I wonder that you could extract proper bounding boxes with your model.

**Comment by doughtmw** [comment]
That's strange that the model isn't working in your setup. I've been using it in the Windows ML API, tested it in CoreML and Keras, and haven't had any issues with detection or proper bounding box outputs. 

As you mentioned, the `reorg` layer isn't well represented in ONNX opset 7 or 8; however, I was able to remove the `reorg` layer and replace it with several `maxpool` layers as suggested in [this comment](https://github.com/allanzelener/YAD2K/issues/80#issuecomment-347211163) and am getting reasonable results. My use-case required ONNX opset 7 or 8, but newer versions of ONNX do not have the same limitations with representing these layers. In the coming weeks, I can create the YOLOv2 ONNX model with the newest ONNX release and add that to this repo if that would be of use.

**Comment by leejaymin** [comment]
@doughtmw Thank you for your thorough explanation. For a few days, I ran your model. I reveal strange points as follows.
First, the output tensor in the last layer (13x13x425) is quite different to the reference output from Darknet. 
![image](https://user-images.githubusercontent.com/1924507/75985361-881e2500-5f2f-11ea-8090-a9184d0392fa.png)
Due to this point, in my case, bounding boxes that are produced by the output tensor are not well represented. 

Second, the weights of ONNX are not equal to Darknet weights. I extracted yolov2 weights from ONNX and then directly compared them to darknet weights. Of course, I arranged the orders of weights along with the sequence of darknet weights. The differences stem from Batch norms.
![image](https://user-images.githubusercontent.com/1924507/75985752-2dd19400-5f30-11ea-8bc2-97b89a2f3e1a.png)

Due to the above reasons, I did not get any bounding boxes after processing Dog.jpg image and Yolov2. 

@prasanthpul Could I make a PR for new yolov2 ONNX? I made it using ONNX opset-9. Also, I verified the weights and output.
